### DISCUSSION TITLE
guide/bar/Java8%E6%96%B0%E7%89%B9%E6%80%A7

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#641] [Announcements]
# guide/bar/Java8%E6%96%B0%E7%89%B9%E6%80%A7

测试1 测试1 测试1 测试1 测试1 测试2 测试3 测试4

http://localhost:8080/guide/bar/Java8%E6%96%B0%E7%89%B9%E6%80%A7.html

**Comment by ww-coding** [comment]
使广大附

**Comment by ww-coding** [comment]
第三方斯蒂芬亲亲我秦伟伟呃呃武器
